products/gstreamer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: GStreamer
+category: framework
+iconSlug: gstreamer
+permalink: /gstreamer
+versionCommand: gst-inspect-1.0 --version
+releasePolicyLink: https://gstreamer.freedesktop.org/
+changelogTemplate: https://gstreamer.freedesktop.org/data/pkg/src/1.11.90/{{"__RELEASE_CYCLE__"}}
```

Comment:
Maybe `https://gstreamer.freedesktop.org/releases/__RELEASE_CYCLE__/` would be more appropriate ?